openfaas/faas-netes
# Bump nat-streaming to 0.11.0

## Description
- Bump the nats-streaming version to 0.11.0 in the docker-compose files, this is the latest tag published on Docker Hub
- Add names to nats ports
- Update nats command to enable the monitoring port



## Motivation and Context


- Addresses openfaas/faas#790

## How Has This Been Tested?



Similar to my testing in openfaas/faas#892

I have started by testing this locally with 

1. create a request callback bin here https://webhook.site and set the bin as a variable 
```sh
export hook=<>
```
2. deploy using the helm chart (and then tested again with the yaml manifest files)
3. deploy the default stack in the faas-cli repo using `faas-cli deploy`
4. testing the functions using sync and async calls, e.g.
```sh
$ echo "$SECONDS" | faas-cli invoke --gateway localhost:31112 -a --header "X-Callback-Url=$hook" stronghash
Function submitted asynchronously.
$ echo "Hi There $SECONDS" | faas-cli invoke --gateway localhost:31112  -a --header "X-Callback-Url=$hook" ruby-echo
Function submitted asynchronously.
```
5. watch the queue worker logs to see it respond
6. check the webhook bin to see the function output 
![image](https://user-images.githubusercontent.com/891889/46427662-4093d800-c742-11e8-94d7-2f2d03979eed.png)


So far, it seems to work as expected for both the helm chart and the manifest files.  I also tested using `nats-top` on the monitoring port to watch as the server processed new events.


## Types of changes


## Checklist:
